games/h.yaml

```diff
@@ -73,12 +73,14 @@
   updated: 2017-10-23
   name: Hedgewars
   development: active
-  lang: [Pascal, C++, Haskell]
+  lang: [Lua, C, C++, Pascal]
+  framework: [SDL, QT]
   repo: http://hg.hedgewars.org/hedgewars/
```

Comment:
Why remove haskell? The gameserver uses it
http://hg.hedgewars.org/hedgewars/file/e461deddc942/gameServer

Reply:
My bad, it uses so many languages and I couldn't see Haskell code anywhere.